README.md

```diff
@@ -3,765 +3,222 @@
-4. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#development-ui) section for more details.
-5. _(Optional)_ Run `make run-ui` (or `make run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:8088> (in production mode). Its HTTP auth credentials are set in `.env-ui`. The UI API also has a frontend for API documentation (default <http://localhost:3002>).
+## Quick Start Installation
+> [!NOTE]
+> The provided installation guide is **non-production** ready. For production deployments please refer to  [Standalone Mode Guide](https://devs.polygonid.com/docs/issuer/setup-issuer-core/).
+>
+> There is no compatibility with Windows environments at this time.
 
-#### Docker Guide Requirements
+### Prerequisites
 
 - Unix-based operating system (e.g. Debian, Arch, Mac OS)
 - [Docker Engine](https://docs.docker.com/engine/) `1.27+`
 - Makefile toolchain `GNU Make 3.81`
+- Publicly accessible URL - The issuer node API must be publicly reachable. Please make sure you properly configure your proxy or use a tool like [Localtunnel](https://theboroer.github.io/localtunnel-www/) for testing purposes.
+- Polygon Mumbai or Main RPC - You can get one in any of the providers of this list
+    - [Chainstack](https://chainstack.com/)
+    - [Ankr](https://ankr.com/)
+    - [QuickNode](https://quicknode.com/)
+    - [Alchemy](https://www.alchemy.com/)
+    - [Infura](https://www.infura.io/)
 
-> **NOTE:** There is no compatibility with Windows environments at this time.
+### Issuer Node Installation
 
-To help expedite a lot of the Docker commands, many have been abstracted using `make` commands. Included in the following sections are the equivalent Docker commands that show what is being run.
+In this section we will cover the installation of the Issuer Node API.
 
-#### Create Docker Configuration Files
+> [!NOTE]
+> This Quick Installation Guide is prepared for Polygon Mumbai (Testnet) both for the state contract and issuer dids. If you want to deploy the node with Polygon Main configuration, please visit our [advanced Issuer Node configuration guide](https://devs.polygonid.com/docs/issuer/issuer-configuration/)).
 
-Make a copy of the following environment variables files:
 
-```bash
-# FROM: ./
+#### Deploy Issuer Node Infrastructure
 
-cp .env-api.sample .env-api;
-cp .env-issuer.sample .env-issuer;
-# (Optional - For issuer UI)
-cp .env-ui.sample .env-ui;
-```
+1. Copy the config sample files:
 
-#### Node Issuer Configuration
+    ```bash
+    cp .env-issuer.sample .env-issuer
+    cp .env-api.sample .env-api
+    ```
 
-The `.env-issuer` will be loaded into the [Docker compose initializer](/infrastructure/local/docker-compose.yml)
+2. Fill the .env-issuer config file with the proper variables:
 
-Any of the following RPC providers can be used:
+   *.env-issuer*
+    ```bash
+    ISSUER_ETHEREUM_URL=<YOUR_RPC_PROVIDER_URI_ENDPOINT>
```

Comment:
Why explicitly mention only this var?

Reply:
that's the one one that MUST be configured for both, API and API-UI and as it is common in both, if it is given a value during the infra process, it is clearer rather having that process 2 times one for each API.